source/uwp/Build/SignConfig.xml

```diff
@@ -15,5 +15,7 @@
     <file src="__RELBINPATH__\..\..\..\s\tosign\AdaptiveCardRenderer\x64\AdaptiveCards.Rendering.Uwp.dll" signType="Authenticode" dest="__RELBINPATH__\..\..\..\s\signed\AdaptiveCardRenderer\x64\AdaptiveCards.Rendering.Uwp.dll" />
     <!-- ARM -->
     <file src="__RELBINPATH__\..\..\..\s\tosign\AdaptiveCardRenderer\ARM\AdaptiveCards.Rendering.Uwp.dll" signType="Authenticode" dest="__RELBINPATH__\..\..\..\s\signed\AdaptiveCardRenderer\ARM\AdaptiveCards.Rendering.Uwp.dll" />
+	<!-- ARM64 -->
```

Comment:
oops! looks like a tab slipped in here :)

Reply:
Fixed.